test/fail_compilation/fail222.d

```diff
@@ -1,9 +1,10 @@
 /*
 TEST_OUTPUT:
 ---
-fail_compilation/fail222.d(10): Error: template `fail222.getMixin(TArg..., int i = 0)()` template tuple parameter must be last one
-fail_compilation/fail222.d(17): Error: template instance `getMixin!()` does not match template declaration `getMixin(TArg..., int i = 0)()`
-fail_compilation/fail222.d(20): Error: template instance `fail222.Thing!()` error instantiating
+fail_compilation/fail222.d(11): Error: template `fail222.getMixin(TArg..., int i = 0)()` template tuple parameter must be last one
```

Comment:
> must be _the_ last one

or 

> must be last

Reply:
I didn't make that message though, it's pre-existing.

Reply:
OK